Shared/Sources/AutoResizingMaskViewSource.swift

```diff
@@ -18,34 +19,51 @@ final class AutoResizingMaskViewSource {
 
   // MARK: Private
 
-  private static let padding: CGFloat = 20
+  private static let innerPadding: CGFloat = 20
   private static let cornerRadius: CGFloat = 2
   private static let borderWidth: CGFloat = 1
-  private static let borderColor = UIColor.whiteColor().colorWithAlphaComponent(0.5).CGColor
+  private static let borderColor = UIColor.borderColor().colorWithAlphaComponent(0.5).CGColor
   private static let defaultViewSideLength: CGFloat = 50
   private static let leftRightWidth: CGFloat = 80
 
-  private let origin: CGPoint
+  private let outerOrigin: CGPoint
+  private let innerOrigin: CGPoint
   private let flexibleWidthHeightSize: CGSize
   private let flexibleTopBottomSize: CGSize
```

Comment:
Considering all the properties that have been added here, I reckon we'd be better off factoring it out in a `struct` model. What do you think?